### ISSUE TITLE
Please update "Additional Requirements" header/link on the Token creation page

### ISSUE THREAD

**Issue by shonuffy** [soonaverse/soonaverse-dao#891]
Please change the "Additional Requirements" section to

Token Sale Requirements
To create a public sale or use the Token Exchange you must complete our KYC process. 

Button text: Begin Process

<img width="1427" alt="Screen Shot 2022-09-29 at 11 34 28 AM" src="https://user-images.githubusercontent.com/80289564/193146349-d299f730-16df-4456-b6f1-4c12f833989f.png">

Also, since we changed the form and are now requiring them to complete the token set first, should we have this form at the end of the set up process rather than the start?

**Comment by adamunchained** [comment]
updated text/title.It'll be in prod within 1h.

Moving it to the end will be a bigger effort for us right now. Let's do it as separate task potentially. 

TBH - they kinda do both in parallel. They don't also have to fill it up if they just want to use our minting function.

**Comment by shonuffy** [comment]
Thanks for the update
